dimod/generators/gates.py

```diff
@@ -68,21 +78,27 @@ def and_gate(in0: Variable, in1: Variable, out: Variable,
 
 def fulladder_gate(in0: Variable, in1: Variable, in2: Variable, sum_: Variable, carry: Variable,
                    *, strength: float = 1.0) -> BinaryQuadraticModel:
-    """Return a binary quadratic model with ground states corresponding to a
+    """Generate a binary quadratic model with ground states corresponding to a
     full adder gate.
 
     Args:
-        in0: The variable label for one of the inputs.
-        in1: The variable label for one of the inputs.
-        in2: The variable label for one of the inputs
-        sum_: The variable label for the sum output.
-        carry: The variable label for the carry output.
-        strength: The energy of the lowest-energy infeasible state.
+        in0: Variable label for one of the inputs.
+        in1: Variable label for one of the inputs.
+        in2: Variable label for one of the inputs
+        sum_: Variable label for the sum output.
+        carry: Variable label for the carry output.
+        strength: Energy of the lowest-energy infeasible state.
 
     Returns:
         A binary quadratic model with ground states corresponding to a full
         adder gate. The model has five variables and ten interactions.
 
+    Examples:
+        >>> bqm = dimod.generators.fulladder_gate('x1', 'x2', 'x3', 'z', 'c')
+        >>> bqm.fix_variables({'x1': 0, 'x2': 1, 'x3': 1})
+        >>> print(dimod.ExactSolver().sample(bqm).first)
```

Comment:
Why not just do the same thing as with `and_gate`?

Reply:
To save space... I can limit the output lines instead